manager/Makefile

```diff
@@ -15,6 +15,11 @@ include $(ROOT_DIR)/hack/make-rules/verify.mk
 pre-test: generate manifests $(TOOLBIN)/etcd $(TOOLBIN)/kube-apiserver
 	mkdir /tmp/taxonomy || true                                                                                                                          
 	cp ../charts/fybrik/files/taxonomy/*.json /tmp/taxonomy/
+	mkdir testdata/unittests/basetaxonomy || true
+	mkdir testdata/unittests/enhancedtaxonomy || true
+	cp ../charts/fybrik/files/taxonomy/*.json testdata/unittests/basetaxonomy
+	cp ../charts/fybrik/files/taxonomy/*.json testdata/unittests/enhancedtaxonomy
```

Comment:
Can you change `enhancedtaxonomy `to `sampletaxonomy` and add the samples for catalog and module when adding the layers?

Reply:
Ok I changed the `taxonomy compile` command to generate a `taxonomy.json` with the layers in `application`, `module`, and `catalog` and saved it in `testdata/unittests/sampletaxonomy`